User-Help/accessibility.md

```diff
@@ -1,6 +1,6 @@
 | URL slug | Category |
 |:--------:|:--------:|
-| accessibility | About the Network |
+| accessibility-policy | About the Network |
```

Comment:
I don't understand why this change is showing here.  It was merged to master months ago and I didn't commit it on this branch.